backend/src/server.ts

```diff
@@ -35,6 +36,14 @@ export const createApp = (options: RouterOptions) => {
     .use(cors())
     .use("/trpc", trpcExpress.createExpressMiddleware(options))
     .use("/ping", (_req, res) => res.send("OK"))
+    .use("/global-state", async (req, res) => {
```

Comment:
This handle is needed to wait in playwright for global state to get populated so we always get success state of subscriptions on pages.